tsdb/engine/tsm1/digest.go

```diff
@@ -22,7 +22,7 @@ type DigestOptions struct {
 
 // DigestWithOptions writes a digest of dir to w using options to filter by
 // time and key range.
-func DigestWithOptions(dir string, files []string, opts DigestOptions, w io.WriteCloser) error {
+func DigestWithOptions(dir string, files []string, opts DigestOptions, w io.WriteCloser) (err error) {
```

Comment:
Do we use digesting for something in 2.x?

Reply:
Hmmm it looks like no (the call chain traces to `ShardDigest`, which isn't called anywhere). Think this should be closed? Or merged to keep things as in-sync as possible while it's easy?